### DISCUSSION TITLE
[Feedback]Acessando o CasaOS de outros computador ou dispositivo.

### DISCUSSION THREAD

**Discussion by OnaySapo** [IceWhaleTech/CasaOS#1217] [Q&A]
### Description

Gostaria de saber como eu poderia acessar meu CasaOS de outros computadores, como em uma casa de um amigo com Windows ou no celular Android.

### Additional Information

	 - OS: linux
	 - CasaOS Version: 0.4.3
	 - Disk Total: 915410 
	 - Disk Used: 15934 
	 - System Info: [BigLinux, based in Manjaro Linux 
 ( ) (\l) Bit:64 Mem:15876M Core:8 Linux onay-dq77pro 6.1.31-2-MANJARO #1 SMP PREEMPT_DYNAMIC Sun Jun 4 12:31:46 UTC 2023 x86_64 GNU/Linux]
	 - Remote Version: 0.4.3
	 - Browser: Chrome 
	 - Version: 114.0.0.0

**Comment by alex-des-santos** [comment]
Opa, cara, acho que o jeito mais fácil e rápido é usando o CloudFlare tunnels. Dá uma olhada nesse vídeo [CloudFlareTunnes](https://www.youtube.com/watch?v=Q5dG8g4-Sx0&t=5s) . Você roda um comando bem simples pra subir um container, e faz todo o gerenciamento pelo site do CloudFlare. O ideal é ter um dominio pra acessar via web.
